DataFormats/PortableTestObjects/test/TestSoA.cc

```diff
@@ -0,0 +1,3 @@
+#include "DataFormats/PortableTestObjects/interface/TestSoA.h"
+
+int main() { return 0; }
```

Comment:
Could you add a comment describing why this test is needed? (I imagine to have a minimal source file compiling the  `TestSoA.h` header)

Would it be useful to e.g. default-construct the `TestSoA` or something so that at least some of the generated classes get actually used?

Reply:
Sure, the easiest approach is to write a test using `TestHostCollection`.